### DISCUSSION TITLE
Hidden Field

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3124] [Feature Requests]
What were we thinking for the "Hidden Field" toggle? It seems to be gone... were we planning on this being permissions? 

If so, permissions won't be enough, as we need a global way to hide things like dividers/aliases on the browse page. Maybe this could be an option on the "How would you like this displayed" Display pane in the Field Setup modal? @rijkvanzanten

**Comment by rijkvanzanten** [comment]
Hidden browse / hidden display has been combined into a single `hidden` flag in the DB, and is managed on that data-model overview page where you also have the grid of fields. Moving a field to _hidden_ in there hides the field

**Reply by benhaynes** [reply]
Ahh, right. So do you think the best way to handle this is to add a flag to the interfaces for "form only" or "hide on browse"? I don't think we need to busy up the Settings with a toggle, and most interfaces will be pretty obvious which don't belong on browse. Mostly this is for dividers, but I don't want to apply it to all aliases, since I see some of those eventually having some "calculated" values.

**Reply by rijkvanzanten** [reply]
I'd rather not do it based on the interface used, but rather based on the special flag in directus_fields. If a field has special=form (name tbd) we hide it on browse

**Reply by benhaynes** [reply]
Agree. Potential names: `form`, `detail`, `presentation`, `appearance`, `facade`, `design` (`layout` and `display` are out since we're using those terms elsewhere).

As we discussed on Discord, I think that `form` is a bit too specific to "CMS" and we want to prepare for more "dashboard" detail pages. `presentation` makes the most sense... but it's so loooooong. `detail` is accurate, but is more specific to our terminology (the "detail page").

**Comment by rijkvanzanten** [comment]
This has since been implemented as a flag on the extension manifest
